servlet/src/test/java/org/togglz/servlet/test/util/HttpServletRequestHolderTest.java

```diff
@@ -24,9 +25,12 @@
     public static WebArchive createDeployment() {
         return Deployments.getBasicWebArchive()
             .addClass(HttpServletRequestHolderServlet.class)
+            .addClass(TogglzFilterServlet.class)
```

Comment:
I'm wondering if the test wouldn't be easier, if we simply have a single Servlet for the test. If `HttpServletRequestHolderServlet` would also output the value of the `EXECUTED` attribute, we would only have a single servlet. The servlet could output something like:

```
Query: number=42
Executed: true
```

Then we just need to adjust the asserts to check for `Query: number=42` instead of just `number=42`.

Reply:
Believe or not, that was exactly how I implemented it the first iteration :smile: 

I wanted to have a better way to test the feature, other than looking at the side effects of the implementation (the request attribute marker), but I could not find a better way without breaking the implied contract of `org.togglz.servlet.CompositeRequestListener`.

Anyway, I'll submit a new version very soon!